source/config-file/specify-connection-options.txt

```diff
@@ -0,0 +1,82 @@
+.. _compass-specify-connection-options:
+
+================================
+Specify Read Preference and Tags
+================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can set required connection option values for when you connect to your 
+MongoDB deployment through |compass-short| with the ``forceConnectionOptions`` 
+option. You can restrict how users can connect to your deployment so their read 
+operations do not impact the availability of replica set primaries.
+
+Procedure
+---------
+
+In this example, you limit read operations to read only from ``SECONDARY`` 
+replica set members with the ``ANALYTICS`` and ``READ_ONLY`` tags. You can 
+specify these options on the command line or in a :ref:`configuration 
+file <config-file>`. 
+
+Command Line Example
+~~~~~~~~~~~~~~~~~~~~
+
+The following command starts |compass| from the command line and uses
+``--forceConnectionOptions`` to set the ``readPreference`` and 
+``readPreferenceTags`` options:
+
+.. code-block:: sh
+
+   <path-to-Compass-executable> \
+   --forceConnectionOptions.readPreference=secondary \
+   --forceConnectionOptions.readPreferenceTags=nodeType:ANALYTICS \
+   --forceConnectionOptions.readPreferenceTags=nodeType:READ_ONLY
+
+.. note::
+
+  The name and filepath of the |compass-short| executable depend on your
+  operating system.
+
+Configuration File Example
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can specify the |compass-short| configuration file in either EJSON
+or YAML format. The following configurations set the
```

Comment:
Consider dropping "following":


To set the ``readPreference`` and ``readPreferenceTags`` options, use ``--forceConnectionOptions``.

or 

To assign values to ``readPreference`` and  ``readPreferenceTags``, use ``--forceConnectionOptions``.